src/stack-hci/azext_stack_hci/tests/latest/test_stack_hci_commands.py

```diff
@@ -107,3 +116,29 @@ def test_stack_hci_extension_crud(self):
             self.check('type', 'microsoft.azurestackhci/clusters/arcsettings/extensions')
         ])
         self.cmd('stack-hci extension delete -n {type} -g {rg} --cluster-name {cluster_name} --arc-setting-name default --no-wait --yes')
+
+    @ResourceGroupPreparer(name_prefix='cli_test_stack_hci_cluster', location='eastus')
+    def test_stack_hci_cluster_identity(self):
+        self.kwargs.update({
+            'cluster_name': self.create_random_name('cluster', 15),
+            'cluster_name1': self.create_random_name('cluster', 15),
+            'app_name': self.create_random_name('app', 15)
+        })
+        self.kwargs['client_id'] = self.cmd('ad app create --display-name {app_name}').get_output_in_json()['appId']
+        self.kwargs['tenant_id'] = self.cmd('account show').get_output_in_json()['tenantId']
+
+        self.cmd('stack-hci cluster create -n {cluster_name} -g {rg} --aad-client-id {client_id} --aad-tenant-id {tenant_id} --tags key0=value0 --mi-system-assigned', checks=[
+            self.check('identity.type', 'SystemAssigned')
+        ])
+        try:
+            self.cmd('stack-hci cluster identity assign --cluster-name {cluster_name} -g {rg} --user-assigned id1')
+        except Exception as ex:
+            self.assertTrue("Resource type 'Microsoft.AzureStackHCI/clusters' does not support creation of 'UserAssigned' resource identity." in str(ex))
```

Comment:
I would like to confirm with you that `StackHCI` currently does not support user assigned identity, which is in line with expectations? If so, can we not add user assigned identity related parameters for now?